src/pages/vault/downloads/index.tsx

```diff
@@ -3,25 +3,33 @@ import { GetStaticProps } from 'next'
 import vaultData from 'data/vault.json'
 import installData from 'data/vault-install.json'
 import { ProductData } from 'types/products'
-import { generateStaticProps, GeneratedProps } from 'lib/fetch-release-data'
 import CoreDevDotLayout from 'layouts/core-dev-dot-layout'
 import ProductDownloadsView from 'views/product-downloads-view'
+import { generateStaticProps } from 'views/product-downloads-view/server'
+import {
+  ProductDownloadsViewStaticProps,
+  RawProductDownloadsViewContent,
+} from 'views/product-downloads-view/types'
 
-const VaultDownloadsPage = (props: GeneratedProps): ReactElement => {
-  const { latestVersion, releases } = props
+const VaultDownloadsPage = ({
+  latestVersion,
+  releases,
+  pageContent,
+}: ProductDownloadsViewStaticProps): ReactElement => {
   return (
     <ProductDownloadsView
       latestVersion={latestVersion}
-      pageContent={installData}
+      pageContent={pageContent}
       releases={releases}
     />
   )
 }
```

Comment:
💭 Do you think we can update these page components to do just `export default ProductDownloadsView` and update `ProductDownloadsView` to set its `layout` to `CoreDevDotLayout`? Not at all required for this PR, just wondering if we want to update it while we're here or not. Totally up to you!

Reply:
Great call, absolutely! ✅ Updated.